examples/qml_features/qml/pages/CustomBaseClassPage.qml

```diff
@@ -9,83 +9,127 @@ import QtQuick.Layouts 1.12
 import com.kdab.cxx_qt.demo 1.0
 
 Page {
+    property var activeModel: tabs.currentIndex == 0 ? customBaseClass : transitiveInheritance
+    property var activeListView: tabs.currentIndex == 0 ? customBaseList : transitiveList
+
     header: ToolBar {
         RowLayout {
             anchors.fill: parent
 
             ToolButton {
                 text: qsTr("Add Row")
 
-                onClicked: customBaseClass.add()
+                onClicked: activeModel.add()
```

Comment:
prefer fully qualified over using context, so add `id: root` then `root.activeModel.add()`

We probably don't do this in all places yet but it's the modern way and what things like qmllint will recommend

Reply:
Done